14.0.Dockerfile

```
@@ -0,0 +1,185 @@
+    WITHOUT_DEMO=True \
+    WAIT_PG=true \
+    PGUSER=odoo \
+    PGPASSWORD=odoo \
+    PGHOST=db \
+    PGPORT=5432 \
+    ADMIN_PASSWORD=admin
+
+# Other requirements and recommendations to run Odoo
+# See https://github.com/$ODOO_SOURCE/blob/$ODOO_VERSION/debian/control
+ARG WKHTMLTOPDF_VERSION=0.12.5
+ARG WKHTMLTOPDF_CHECKSUM='1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb'
+RUN apt-get -qq update \
+    && apt-get install -yqq --no-install-recommends \
+        curl \
+    && curl -SLo wkhtmltox.deb https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox_${WKHTMLTOPDF_VERSION}-1.stretch_amd64.deb \
+    && echo "${WKHTMLTOPDF_CHECKSUM}  wkhtmltox.deb" | sha256sum -c - \
+    && apt-get install -yqq --no-install-recommends \
+        ./wkhtmltox.deb \
+        chromium \
+        ffmpeg \
+        fonts-liberation2 \
+        gettext-base \
+        git \
+        gnupg2 \
+        locales-all \
+        nano \
+        npm \
+        wget \
+        openssh-client \
+        telnet \
+        vim \
+        zlibc \
+        sudo \
+    && echo 'deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main' >> /etc/apt/sources.list.d/postgresql.list \
+    && curl -SL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+    && apt-get update \
+    && apt-get install -yqq --no-install-recommends postgresql-client \
+    && apt-get autopurge -yqq \
+    && rm -Rf wkhtmltox.deb /var/lib/apt/lists/* /tmp/* \
+    && sync
+
+ARG ODOO_VERSION=13.0
+ARG ODOO_SOURCE=odoo/odoo
+ARG ODOO_SOURCE_DEPTH=1
+ENV ODOO_VERSION="$ODOO_VERSION"
+ENV ODOO_SOURCE="$ODOO_SOURCE"
+ENV ODOO_SOURCE_DEPTH="$ODOO_SOURCE_DEPTH"
+
+# Install Odoo hard & soft dependencies, and Doodba utilities
+RUN build_deps=" \
+        build-essential \
+        libfreetype6-dev \
+        libfribidi-dev \
+        libghc-zlib-dev \
+        libharfbuzz-dev \
+        libjpeg-dev \
+        liblcms2-dev \
+        libldap2-dev \
+        libopenjp2-7-dev \
+        libpq-dev \
+        libsasl2-dev \
+        libtiff5-dev \
+        libwebp-dev \
+        tcl-dev \
+        tk-dev \
+        zlib1g-dev \
+    " \
+    && apt-get update \
+    && apt-get install -yqq --no-install-recommends $build_deps \
+    && pip install --no-cache-dir \
+        -r https://raw.githubusercontent.com/$ODOO_SOURCE/$ODOO_VERSION/requirements.txt \
+        git-aggregator \
+        ipython \
+        pysnooper \
+        ipdb \
+        git+git://github.com/OCA/openupgradelib.git \
+        click-odoo-contrib \
+        geoip2 \
+        pg_activity \
+        phonenumbers \
+    && (python3 -m compileall -q /usr/local/lib/python3.7/ || true) \
+    && apt-get purge -yqq $build_deps \
+    && apt-get autopurge -yqq \
+    && rm -Rf /var/lib/apt/lists/* /tmp/*
+
+# Metadata
+ARG VCS_REF
+ARG BUILD_DATE
+ARG VERSION
+LABEL org.label-schema.schema-version="$VERSION" \
+      org.label-schema.vendor=Druidoo \
```

```suggestion
      org.label-schema.vendor=Druidoo \
```

:thinking: esto debe haber quedado de cuando inicializamos el repo, capaz queres cambiarlo en las otras tmb

---

perfecto, que decís, las borramos directamente? o parametrizables? yo no las uso para nada, por mi borramos..

---

como quieras.. no se usan. pero por ahi querias ponerle ingadhoc ya que esta en este repo